modules/misc/doc/content/source/auxkernels/CoupledDirectionalMeshHeightInterpolation.md

```diff
@@ -1 +1,15 @@
-!template load file=stubs/moose_object.md.template name=CoupledDirectionalMeshHeightInterpolation syntax=/AuxKernels/CoupledDirectionalMeshHeightInterpolation
\ No newline at end of file
+# CoupledDirectionalMeshHeightInterpolation
+
+!syntax description /AuxKernels/CoupledDirectionalMeshHeightInterpolation
+
+## Description
+
+This object scales a user-provied coupled variable (either a solution variable or an auxiliary variable) by its position relative to the maximum and minimum coordinate of the undeformed mesh in a user-specified direction. When the coordinate is equal to the minimum coordinate, the scaling factor is 0, while when it is at the maximum coordinate, the scaling factor is 1, the scaling factor is linearly interpolated between those bounding values.
```

Comment:
Whoops, missed a period? `the scaling factor is 1. The scaling factor is linearly interpolated between those bounding values.`

Reply:
Good catch! I just revised that last sentence.